Taskfile.yml

```diff
@@ -234,7 +239,7 @@ tasks:
     dir: tdrs-frontend
     cmds:
       - docker-compose -f docker-compose.local.yml up -d tdp-frontend-test --quiet-pull
-      - docker-compose -f docker-compose.yml exec tdp-frontend-test sh -c "npm run lint"
+      - docker-compose -f docker-compose.yml exec -T tdp-frontend-test sh -c "npm run lint"
```

Comment:
```suggestion
      - docker compose -f docker-compose.yml exec -T tdp-frontend-test sh -c "npm run lint"
```

Reply:
Unfortunately, these changes break my local use of Taskfile. We'll need to investigate your docker compose config so we're all on the same page. Additionally, just these 3 changes don't address what's needed for all other Tasks to run. Let's sync up when you're available

Reply:
```bash
$ docker -v
Docker version 27.1.1, build 63125853e3
```